ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -750,6 +755,10 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
             );
         }
 
```

Comment:
need to refactor logic here

```rust
for r_table in cs.r_table_expressions.iter() {
            match &r_table.table_spec.addr_type {
                SetTableAddrType::FixedAddr => (),
                SetTableAddrType::DynamicAddr(spec) => {
                    let expected_eval = eval_wellform_address_vec(
                        spec.offset as u64,
                        WORD_SIZE as u64,
                        &input_opening_point,
                    );
                    if expected_eval != proof.wits_in_evals[spec.addr_witin_id] {
                        return Err(ZKVMError::VerifyError(
                            "dynamic addr evaluate != expected_evals".into(),
                        ));
                    }
                }
            }
        }
```
by iterating through structral_witness witin instead of table spec